Traceback (most recent call last):
  File "/Volumes/zorg/mb312/.virtualenvs/lis-lession/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Volumes/zorg/mb312/.virtualenvs/lis-lession/lib/python3.8/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Volumes/zorg/mb312/.virtualenvs/lis-lession/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Volumes/zorg/mb312/.virtualenvs/lis-lession/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/local/opt/python@3.8/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Volumes/zorg/mb312/.virtualenvs/lis-lession/lib/python3.8/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Volumes/zorg/mb312/.virtualenvs/lis-lession/lib/python3.8/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Volumes/zorg/mb312/.virtualenvs/lis-lession/lib/python3.8/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# A map of Oakland and the Bay Area.
oak_map = folium.Map(location=[37.8, -122.4],
tiles='OpenStreetMap',
zoom_start=10)

# Mark Police Department
OAK_PD_COORDS = (37.79926910804871, -122.27497697421084)
folium.Marker(OAK_PD_COORDS,
              popup='Police Dept',
              icon=folium.Icon(color='lightgray', icon='info-sign')
             ).add_to(oak_map)


# Mark City Hall
OAK_CITY_HALL_COORDS = (37.805498393033204, -122.2725551318817)
folium.Marker(OAK_CITY_CALL_COORDS,
              popup='City Hall',
              icon=folium.Icon(color='lightgray', icon='info-sign')
             ).add_to(oak_map)

# Show the map
oak_map
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Input [0;32mIn [2][0m, in [0;36m<cell line: 16>[0;34m()[0m
[1;32m     14[0m [38;5;66;03m# Mark City Hall[39;00m
[1;32m     15[0m OAK_CITY_HALL_COORDS [38;5;241m=[39m ([38;5;241m37.805498393033204[39m, [38;5;241m-[39m[38;5;241m122.2725551318817[39m)
[0;32m---> 16[0m folium[38;5;241m.[39mMarker([43mOAK_CITY_CALL_COORDS[49m,
[1;32m     17[0m               popup[38;5;241m=[39m[38;5;124m'[39m[38;5;124mCity Hall[39m[38;5;124m'[39m,
[1;32m     18[0m               icon[38;5;241m=[39mfolium[38;5;241m.[39mIcon(color[38;5;241m=[39m[38;5;124m'[39m[38;5;124mlightgray[39m[38;5;124m'[39m, icon[38;5;241m=[39m[38;5;124m'[39m[38;5;124minfo-sign[39m[38;5;124m'[39m)
[1;32m     19[0m              )[38;5;241m.[39madd_to(oak_map)
[1;32m     21[0m [38;5;66;03m# Show the map[39;00m
[1;32m     22[0m oak_map

[0;31mNameError[0m: name 'OAK_CITY_CALL_COORDS' is not defined
NameError: name 'OAK_CITY_CALL_COORDS' is not defined

